lib/copy-sync/__tests__/copy-sync-dir.test.js

```diff
@@ -147,50 +146,47 @@ describe('+ copySync()', () => {
 
       assert(!fs.existsSync(path.join(dest, IGNORE)), 'directory was not ignored')
       assert(!fs.existsSync(path.join(dest, IGNORE, 'file')), 'file was not ignored')
-      done()
     })
 
-    it('should apply filter when it is applied only to dest', done => {
+    it('should apply filter when it is applied only to dest', () => {
       const timeCond = new Date().getTime()
 
       const filter = (s, d) => fs.statSync(d).birthtime.getTime() < timeCond
 
-      const subdir = path.join(src, 'subdir')
-      fs.mkdirsSync(subdir)
-
       const dest = path.join(TEST_DIR, 'dest')
 
       setTimeout(() => {
+        fs.mkdirsSync(src)
+        fs.writeFileSync(path.join(src, 'somefile.html'), 'some data')
         fs.mkdirsSync(dest)
         try {
           fs.copySync(src, dest, filter)
         } catch (err) {
           assert.ifError(err)
         }
-        assert(!fs.existsSync(path.join(dest, 'subdir')))
-        done()
```

Comment:
You still need `done` when you're using setTimeout (since that makes the code async).